app/src/main/res/layout/fragment_contact.xml

```diff
@@ -1,12 +1,11 @@
-<android.support.v4.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-        <android.support.v7.widget.RecyclerView
-            android:id="@+id/rv_employees"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:background="@color/white" />
+    <android.support.v7.widget.RecyclerView
```

Comment:
do we need the frame layout? if we only need the recycler view is ok to leave it alone as it will be contained on Activities FrameLayout